app/cli/src/main/scala/org/bitcoins/cli/ConsoleCli.scala

```diff
@@ -1015,6 +1015,56 @@ object ConsoleCli {
                 case other => other
               }))
         ),
+      cmd("createrangedevent")
+        .action((_, conf) =>
+          conf.copy(command = CreateRangedEvent("", Instant.MIN, 0, 0, 1)))
+        .text("Registers an oracle event with a range of outcomes")
+        .children(
+          arg[String]("label")
```

Comment:
any reason to use "label" here rather than "eventName"? I would much prefer "eventName".

Reply:
That is what was on the other one, can fix